bindinfo/handle.go

```diff
@@ -600,7 +600,7 @@ func (h *BindHandle) CaptureBaselines() {
 	for i := range sqls {
 		stmt, err := parser4Capture.ParseOneStmt(sqls[i], "", "")
 		if err != nil {
-			logutil.BgLogger().Debug("parse SQL failed", zap.String("SQL", sqls[i]), zap.Error(err))
+			logutil.BgLogger().Debug("[sql-bind] parse SQL failed in baseline capture", zap.String("SQL", sqls[i]), zap.Error(err))
```

Comment:
I suggest using the error level here since no user uses the debug level in the production environment. 
If these errors occur in the production environment, there is no log left then and it's hard for us to investigate.

Reply:
I changed all the log levels to DEBUG for SQL bind, because SQL bind is a background job, there would be potential log flooding if a query went wrong, just as the referenced issue, users may do not want to see those logs when their bindings are actually working well. If users observe abnormality of SQL bind, DBA can turn on the DEBUG logs for investigation? @SunRunAway @nullnotnil any thoughts on the log level?